frontend/src/pages/Graphing/components/Graph.tsx

```diff
@@ -497,7 +497,7 @@ const Graph = ({
 
 		const newStartFetch = presetStartDate(selectedPreset)
 		const newPollInterval =
-			moment().diff(newStartFetch, 'hours') > 5
+			moment().diff(newStartFetch, 'hours') >= 4
```

Comment:
i dont think this will help with navigating between `/logs` and `/traces` since the time range is always reset to last 15 minutes when you click the respective product link from the header. does this only affect the background polling?